evap/staff/views.py

```diff
@@ -1140,6 +1147,10 @@ def user_import(request):
 
 @staff_required
 def user_edit(request, user_id):
+    @receiver(RewardPointGranting.granted_by_removal)
+    def notify_reward_points(users, **kwargs):
+        messages.info(request, _("The removal of courses has granted {} reward points for the active semester.".format(users[0])))
```

Comment:
can you please change this to `'The removal of courses has granted the user "{}" reward points for the active semester.'`?

Reply:
Sorry for nitpicking, but wouldn't it be `"The removal of courses has granted the user \"{}\" reward points for the active semester."`? Also note that the message makes an assumption, that the form can only edit the active semester and therefore the message's content reflect this.

Reply:
i saw the active semester, that's fine. in some cases we're escaping the double quotation marks, in others we're using single quotation marks for the string when having doubles in there - i don't really care.